Content.Server/UserInterface/ActivatableUISystem.cs

```diff
@@ -0,0 +1,143 @@
+using System.Linq;
+using Content.Shared;
+using Content.Shared.CCVar;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Hands;
+using Content.Shared.Popups;
+using Content.Shared.Standing;
+using Content.Shared.Stunnable;
+using Content.Shared.Throwing;
+using Content.Shared.Interaction;
+using Content.Shared.Interaction.Helpers;
+using Content.Server.Administration.Managers;
+using JetBrains.Annotations;
+using Robust.Server.GameObjects;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Localization;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Network;
+using Robust.Shared.IoC;
+
+namespace Content.Server.UserInterface
+{
+    [UsedImplicitly]
+    internal sealed class ActivatableUISystem : EntitySystem
```

Comment:
In addition to all the duplicate "interact to open" code, there's also several duplicate "add toggle UI verb" code that could be removed. For example, see `GasTankSystem`, `DisposalTubeSystem`, and `StorageSystem`

So it would be nice if this system also supported open/close UI verbs. But this could easily just be done in a future PR, and I'd be happy to do it myself if you don't want to.

Reply:
In the lack of other requested changes, I'd do it, but right now I'm just counting this as out-of-scope for now